### PR TITLE
Section on Azure AD Group Sync

### PR DESCRIPTION
**Pull request by smirman**

### CODE REVIEW

File: docs/idp/az-ad-grp-sync/README.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Using Group Sync Operator with Azure Active Directory and ROSA #
```
Comment (michaelryanmcneill) [inline docs/idp/az-ad-grp-sync/README.md:1]: Can we change this to and ROSA/OSD?

File: docs/idp/az-ad-grp-sync/README.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+1. Validate that all users specified in Azure AD also show up as members of the associated group in ROSA
+
+    ![Users added](./images/grp-sync-success-usr.png)
+
+1. Add a new user in Azure AD and assign it to the admin group
+
+    ![New User added](./images/grp-sync-new-usr.png)
+
+1. Verify that the user now appears in ROSA (after the specified synchronization time)
```
Comment (michaelryanmcneill) [inline docs/idp/az-ad-grp-sync/README.md:117]: Same here ROSA/OSD.

File: docs/idp/az-ad-grp-sync/README.md [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+- Before testing the synchronization, ensure that your Registered Azure Application has permissions for `Group.ReadAll`, `GroupMember.ReadAll`, and `User.ReadAll`
+
+    ![API permissions](./images/grp-sync-api-perm.png)
+
+- Additionally, check to see if the Group Sync process has completed with a `Condition: ReconcileSuccess` message
+
+    ![Successful Sync](./images/grp-sync-success.png)
+
+1. Check to see that all the groups specified in the configuration YAML file show up in the ROSA Groups list
```
Comment (michaelryanmcneill) [inline docs/idp/az-ad-grp-sync/README.md:105]: Same here ROSA/OSD.

File: docs/idp/az-ad-grp-sync/README.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+## Define groups and assign users in Azure AD ##
+
+To synchronize groups and users with ROSA they must exist in Azure AD
+
+1. Create groups to syncronize with ROSA if they do not already exist
+
+    ![Azure AD Groups](./images/az-ad-grp.png)
+
+1. Create user IDs to synchronize with ROSA if they do not already exist
```
Comment (michaelryanmcneill) [inline docs/idp/az-ad-grp-sync/README.md:25]: Same here ROSA/OSD.

File: docs/idp/az-ad-grp-sync/README.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+1. Install the Group Sync Operator from the OpenShift Operator Hub
+1. Create and configure a new Group Sync instance
+1. Set a synchronization schedule
+1. Testing the synchronization process
+
+## Define groups and assign users in Azure AD ##
+
+To synchronize groups and users with ROSA they must exist in Azure AD
+
+1. Create groups to syncronize with ROSA if they do not already exist
```
Comment (michaelryanmcneill) [inline docs/idp/az-ad-grp-sync/README.md:21]: Same here ROSA/OSD.

File: docs/idp/az-ad-grp-sync/README.md [hunk 6]
```diff
@@ -0,0 +1,127 @@
...
+
+1. Define groups and assign users in Azure AD
+1. Install the Group Sync Operator from the OpenShift Operator Hub
+1. Create and configure a new Group Sync instance
+1. Set a synchronization schedule
+1. Testing the synchronization process
+
+## Define groups and assign users in Azure AD ##
+
+To synchronize groups and users with ROSA they must exist in Azure AD
```
Comment (michaelryanmcneill) [inline docs/idp/az-ad-grp-sync/README.md:19]: Same here ROSA/OSD.

File: docs/idp/az-ad-grp-sync/README.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+**Steve Mirman**
+
+*8 November 2021*
+
+This guide focuses on how to synchronize Identity Provider (IDP) groups and users after configuring authentication in OpenShift Cluster Manager (OCM). For an IDP configuration example, please reference the [Configure Azure AD as an OIDC identity provider for ROSA/OSD](https://mobb.ninja/docs/idp/azuread/) guide.
+
+To set up group synchronization from Azure Active Directory (AD) to ROSA you must:
+
+1. Define groups and assign users in Azure AD
+1. Install the Group Sync Operator from the OpenShift Operator Hub
```
Comment (michaelryanmcneill) [inline docs/idp/az-ad-grp-sync/README.md:12]: OperatorHub*

**Review by michaelryanmcneill — CHANGES_REQUESTED** [review]

File: docs/idp/az-ad-grp-sync/README.md (hunk 1 above)
Comment (smirman) [reply docs/idp/az-ad-grp-sync/README.md]: fixed

File: docs/idp/az-ad-grp-sync/README.md (hunk 2 above)
Comment (smirman) [reply docs/idp/az-ad-grp-sync/README.md]: fixed

File: docs/idp/az-ad-grp-sync/README.md (hunk 3 above)
Comment (smirman) [reply docs/idp/az-ad-grp-sync/README.md]: fixed

File: docs/idp/az-ad-grp-sync/README.md (hunk 4 above)
Comment (smirman) [reply docs/idp/az-ad-grp-sync/README.md]: fixed

File: docs/idp/az-ad-grp-sync/README.md (hunk 5 above)
Comment (smirman) [reply docs/idp/az-ad-grp-sync/README.md]: fixed

File: docs/idp/az-ad-grp-sync/README.md (hunk 6 above)
Comment (smirman) [reply docs/idp/az-ad-grp-sync/README.md]: fixed

**Review by michaelryanmcneill — APPROVED** [review]
LGTM

**Pull request merged**